ReadMe.md

```diff
@@ -148,6 +148,10 @@ You can now run the various Run/Debug Configurations such as
 * All Compiler Tests
 * All IDEA Plugin Tests
 
+### Running specific generated tests
+
+If you need to debug a specific generated test ensure that you have the `Working directory` your IntellJ run configuration set
+to the root directory of this project. If you don't every test you try to run will fail with a `No such file or directory` exception.
```

Comment:
Basically change the test run config from `$MODULE_DIR$` to `$PROJECT_DIR$`, right?
Could we maybe fix that, instead of explaining, i.e. by setting the `JUnit` default to it?

Reply:
Hmm, the default seems to be stored in the `workspace.xml`, as
```xml
<option name="WORKING_DIRECTORY" value="%MODULE_WORKING_DIR%" />
```

Changing that to
```xml
<option name="WORKING_DIRECTORY" value="$PROJECT_DIR$" />
```
also solves the problem.

Is there a way to fix that automatically?

Reply:
I'm not sure. Do you have a suggestion?

Reply:
No, just questions :)
Maybe we can ask someone from the IntelliJ team on whether it can be fixed properly.